### DISCUSSION TITLE
How to remove a library from VCPKG that can no longer be supported?

### DISCUSSION THREAD

**Discussion by bobk2824** [microsoft/vcpkg#36756] [Q&A] [answered]
I need to remove a library from VCPKG. It is no longer used internally and won't be supported.

Thanks,
Bob

**Comment by Osyotr** [comment, marked as answer]
If you want to remove locally, use `vcpkg remove my-library` or just remove it from your manifest file.
If you are maintainer of a library and want to remove it from vcpkg registry (i.e. this repo), use this PR as an example: #34419

**Reply by bobk2824** [reply]
The latter. Thanks.
